templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -65,7 +66,6 @@ Metadata:
         Parameters:
             - TomcatContextPath
             - CatalinaOpts
-            - JvmHeapOverride
```

Comment:
We are already at `60` with our Jira param count. Removing this param for now but a larger discussion needs to be had on how best we can handle this.